index.js

```diff
@@ -0,0 +1,106 @@
+import {canvas, ctx, resizeCanvas, renderPaths, drawLine, drawRectangle, drawCircle, undo, renderSquare, renderCircle, finishDrawing} from './functions'
```

Comment:
looks way too long, you can import everything like this:

```
import * as helpers from './functions'
import { canvas, ctx } from './functions'
```

and later you can use it like this:
```
helpers.resizeCanvas
helpers.renderPaths
```

etc.